docs/_data/table/en/user-configuration.yml

```diff
@@ -85,3 +85,5 @@ alluxio.user.ufs.file.writer.class:
   Selects networking stack to run the client with for writing to under file system through a
   worker's data server. Currently only `alluxio.client.netty.NettyUnderFileSystemFileWriter`
   (remote write using netty) is valid.
+alluxio.user.packet.streaming.enabled:
+  If set to true, the packet streaming data transfer protocol is used.
```

Comment:
Could you label this as experimental and add a note about the possible benefits?